tests/models/git/test_modeling_git.py

```diff
@@ -399,7 +399,11 @@ class GitModelTest(ModelTesterMixin, GenerationTesterMixin, PipelineTesterMixin,
     all_model_classes = (GitModel, GitForCausalLM) if is_torch_available() else ()
     all_generative_model_classes = (GitForCausalLM,) if is_torch_available() else ()
     pipeline_model_mapping = (
-        {"feature-extraction": GitModel, "image-to-text": GitForCausalLM, "text-generation": GitForCausalLM}
+        {
+            "feature-extraction": GitModel,
+            "image-to-text": GitForCausalLM,
+            "text-generation": GitForCausalLM,
+        }
```

Comment:
nit: wondering if you manually change this. It doesn't look our styling tool change the previous format.

Reply:
I think it was probably a trailing comma on my part. I've updated to be one line again